src/test/groovy/org/prebid/server/functional/tests/MetricsSpec.groovy

```diff
@@ -0,0 +1,84 @@
+package org.prebid.server.functional.tests
+
+import org.prebid.server.functional.model.config.AccountConfig
+import org.prebid.server.functional.model.config.AccountMetricsConfig
+import org.prebid.server.functional.model.db.Account
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.response.auction.BidResponse
+
+import static org.prebid.server.functional.model.config.AccountMetricsVerbosityLevel.BASIC
+import static org.prebid.server.functional.model.config.AccountMetricsVerbosityLevel.DETAILED
+import static org.prebid.server.functional.model.config.AccountMetricsVerbosityLevel.NONE
+
+class MetricsSpec extends BaseSpec {
```

Comment:
I recommend flushing the metrics before you enter this spec just to be sure that nothing will dirty your tests.